specification/confidentialledger/resource-manager/Microsoft.ConfidentialLedger/preview/2023-06-28-preview/managedccf.json

```diff
@@ -0,0 +1,689 @@
+      "title": "ManagedCCFProperties",
+      "properties": {
+        "appName": {
+          "readOnly": true,
+          "type": "string",
+          "description": "Unique name for the Managed CCF."
+        },
+        "appUri": {
+          "type": "string",
+          "readOnly": true,
+          "description": "Endpoint for calling Managed CCF Service."
+        },
+        "identityServiceUri": {
+          "type": "string",
+          "readOnly": true,
+          "description": "Endpoint for accessing network identity."
+        },
+        "memberIdentityCertificates": {
+          "type": "array",
+          "description": "List of member identity certificates for  Managed CCF",
+          "items": {
+            "$ref": "#/definitions/MemberIdentityCertificate"
+          },
+          "x-ms-identifiers": []
+        },
+        "deploymentType": {
+          "description": "Deployment Type of Managed CCF",
+          "$ref": "#/definitions/DeploymentType"
+        },
+        "runningState": {
+          "description": "Object representing RunningState for Managed CCF.",
+          "$ref": "#/definitions/RunningState"
+        },
+        "provisioningState": {
+          "description": "Provisioning state of Managed CCF Resource",
+          "readOnly": true,
+          "$ref": "#/definitions/ProvisioningState"
+        },
+        "nodeCount": {
+          "type": "integer",
+          "format": "int32",
+          "description": "Number of CCF nodes in the Managed CCF.",
+          "$ref": "#/definitions/NodeCount"
+        }
+      }
+    },
+    "ManagedCCF": {
+      "type": "object",
+      "description": "Managed CCF. Contains the properties of Managed CCF Resource.",
+      "allOf": [
+        {
+          "$ref": "../../../../../common-types/resource-management/v3/types.json#/definitions/TrackedResource"
+        }
+      ],
+      "properties": {
+        "properties": {
+          "description": "Properties of Managed CCF Resource.",
+          "$ref": "#/definitions/ManagedCCFProperties"
+        }
+      }
+    },
+    "ManagedCCFList": {
+      "description": "Object that includes an array of Managed CCF and a possible link for next set.",
+      "type": "object",
+      "properties": {
+        "value": {
+          "description": "List of Managed CCF",
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/ManagedCCF"
+          }
+        },
+        "nextLink": {
+          "description": "The URL the client should use to fetch the next page (per server side paging).",
+          "type": "string"
+        }
+      }
+    },
+    "ManagedCCFBackup": {
+      "description": "Object representing Backup properties of Managed CCF Resource.",
+      "type": "object",
+      "properties": {
+        "restoreRegion": {
+          "type": "string",
+          "readOnly": false,
+          "description": "The region where the backup of the Managed CCF Resource will eventually be restored to."
```

Comment:
>restored [](http://example.com/codeflow?start=101&length=8)

again, why do they supposedly know this in advance at backup time? what if they want to restore to a different region years later?

Reply:
Please take a look at our sample docs for instructions we are providing to restore our service: https://review.learn.microsoft.com/en-us/azure/managed-ccf/how-to-backup-restore-resource?branch=pr-en-us-251133#restore. Our service requires a few resources to be created to the specified restore region in advance. This is why the future restore region is required. Our backups have a 90 days expiry and if the customer would like to restore to a different region they can create a backup for each region of their choosing.